package.json

```diff
@@ -27,6 +30,12 @@
     "styled-components": "^5.3.3"
   },
   "devDependencies": {
+    "@commitlint/cli": "^16.0.1",
+    "@commitlint/config-angular": "^16.0.0",
+    "@jscutlery/semver": "^2.18.0",
+    "@nrwl/cli": "latest",
```

Comment:
Aren't there stable releases for this apps?

Reply:
this is what their installation script added, but yeah makes sense to pin it